README.md

```diff
@@ -215,6 +215,21 @@ class FormIntervalValidation(BaseValidation):
         validate_interval(interval['interval'])
 ```
 
+## VisibilityClass
+
+Make sure, that caluma-interval has access to `Forms`, `Documents` and `Cases`.
+
+```python
+class MyVisibility(BaseVisibility):
+    @filter_queryset_for(Node)
+    def filter_queryset(self, node, queryset, info):
+        if info.context.META.get(
+            "HTTP_CALUMA_COMPANION", None
+        ) == "interval" and node in [Form, Document, Case]:
```

Comment:
I'm not sure we should publish an insecure example like this. If someone copy/pastes this in production, anybody could put a header like this in their requests and see everything.

Instead, I'd do proper auth and check the user's role. Or alternatively, implement a preshared-secret type auth